src/interfaces/src/plugin/api/interfaces/load.h

```diff
@@ -1,9 +1,38 @@
 #ifndef INTERFACES_PLUGIN_API_INTERFACES_LOAD_H
 #define INTERFACES_PLUGIN_API_INTERFACES_LOAD_H
 
+#include "plugin/data/interfaces/interface/hash.h"
+#include "plugin/data/interfaces/interface/linked_list.h"
 #include "plugin/context.h"
-#include <utarray.h>
 
-int interfaces_load_interface(interfaces_ctx_t* ctx, interfaces_interface_hash_element_t** if_hash);
+enum interfaces_load_exit_status {
```

Comment:
All static functions should be put inside the load.c file, header should only define API which can be used in other files (startup/load.c). Enum should also be inside the load.c file.